.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: |
           conda create --yes -n tester python=${{ matrix.python-version }} pip numpy 'scipy>=0.17.0' matplotlib pandas flake8 pep8 jupyter coverage cython scikit-learn requests
           conda install -n tester -c conda-forge phantomjs --yes
-          conda run -n tester pip install sphinx numpydoc sphinx-bootstrap-theme
+          conda run -n tester pip install "sphinx<4" numpydoc sphinx-bootstrap-theme
```

Comment:
```suggestion
          conda run -n tester "pip install 'sphinx<4' numpydoc sphinx-bootstrap-theme"
```

Reply:
@thermokarst I forget how to properly escape the less than in this case so I've gone for my best guess :)